docs/resources/aws_guardduty_detector.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_guardduty_detector Resource
+platform: aws
+---
+
+# Requirements
+
+Train AWS version 0.1.18 or newer is required for this resource.
+
+# aws\_ami
+
+Use the `aws_guardduty_detector` InSpec audit resource to test properties of a single AWS GuardDuty Detector.
+
+## Syntax
+
+An `aws_guardduty_detector` resource block declares the tests for a single AWS GuardDuty Detector by detector id.
+
+    describe aws_guardduty_detector(detector_id: '12abc34d567e8fa901bc2d34e56789f0') do
+      it { should exist }
+    end
+
+#### Parameters
+
+##### detector\_id _(required)_
+
+This resource accepts a single parameter, the GuardDuty Detector ID.
+This can be passed either as a string or as a `detector_id: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on GuardDuty Detectors](https://docs.aws.amazon.com/guardduty/latest/ug/what-is-guardduty.html).
+
+
+## Properties
+
+|Property                       | Description|
+| ---                           | --- |
+|detector\_id                   | The unique ID of the detector that you want to get. |
```

Comment:
Can't see detector id here: 
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/GuardDuty/Client.html#list_detectors-instance_method

Can you double-check if this property exists?

Reply:
Now removed